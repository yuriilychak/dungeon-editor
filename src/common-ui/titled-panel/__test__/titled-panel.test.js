import React from "react";
import { createMount } from "@material-ui/core/test-utils";
import TitledPanel from "../titled-panel";

describe("TitledPanel test",()=> {
    it('default snapshot', () => {
        const wrapper = createMount()(
            <TitledPanel
                title="test"
            >
                Test text
            </TitledPanel>
        );
        expect(wrapper.html()).toMatchSnapshot();
    });

    it('with title children snapshot', () => {
        const wrapper = createMount()(
            <TitledPanel
                title="test"
                titleChildren="Title children"
            >
                Test text
            </TitledPanel>
        );
        expect(wrapper.html()).toMatchSnapshot();
    });
});
