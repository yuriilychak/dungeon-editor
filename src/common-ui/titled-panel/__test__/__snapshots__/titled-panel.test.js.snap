// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitledPanel test default snapshot 1`] = `"<div class=\\"TitledPanel-root-1\\"><div class=\\"TitledPanel-titleContainer-2\\"><span class=\\"MuiTypography-root-5 MuiTypography-button-16 TitledPanel-titleText-4\\">test</span></div><div class=\\"TitledPanel-body-3\\">Test text</div></div>"`;

exports[`TitledPanel test with title children snapshot 1`] = `"<div class=\\"TitledPanel-root-1\\"><div class=\\"TitledPanel-titleContainer-2\\"><span class=\\"MuiTypography-root-5 MuiTypography-button-16 TitledPanel-titleText-4\\">test</span>Title children</div><div class=\\"TitledPanel-body-3\\">Test text</div></div>"`;
